osu.Game/Screens/OnlinePlay/DrawableRoomPlaylist.cs

```diff
@@ -165,7 +165,11 @@ public bool ShowItemOwners
         {
             d.SelectedItem.BindTarget = SelectedItem;
             d.RequestDeletion = i => RequestDeletion?.Invoke(i);
-            d.RequestResults = i => RequestResults?.Invoke(i);
+            d.RequestResults = i =>
+            {
+                SelectedItem.Value = i;
+                RequestResults?.Invoke(i);
```

Comment:
Any reason why this is applied locally to this specific button and not the others?

Reply:
If the item isn't selected the results screen won't display the correct item 😅 .

Arguably it could be moved out into the invocation.

Reply:
I mean sure but if the results button is going to select the item, then why not edit? (I suppose delete doesn't matter as the item is going to be... well, deleted, anyway).

Reply:
I didn't actually check whether edit also required it. Might be best to select for conformity though, I agree.

Reply:
I'm actually not going to do that because download doesn't select so it's all over the place right now.